src/synapse/azext_synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -301,7 +370,7 @@ def _create_workspace(self):
 
         # create synapse workspace
         self.cmd(
-            'az synapse workspace create --name {workspace} --resource-group {rg} --account-url {account-url} '
+            'az synapse workspace create --name {workspace} --resource-group {rg} --storage-account {account-url} '

```

Comment:
Seems it is not a storage account resource id.

Reply:
updated we supported both storage account name and storage resource id.

---
In reply to: [424958763](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r424958763) [](ancestors = 424958763)

Reply:
Have you tested storage account resource id?

---
In reply to: [425052797](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r425052797) [](ancestors = 425052797,424958763)